collect_app/src/test/java/org/odk/collect/android/utilities/FileUtilsTest.java

```diff
@@ -276,4 +276,13 @@ public void mediaDirNameIsCorrect() {
         assertThat(metadataFromFormDefinition.get(FileUtils.FORMID), is("set-geopoint-before"));
         assertThat(metadataFromFormDefinition.get(FileUtils.GEOMETRY_XPATH), is("/data/location1"));
     }
+
+    @Test
+    @SuppressWarnings("PMD.DoNotHardCodeSDCard")
+    public void simplifyScopedStoragePathTest() {
```

Comment:
👍. I had always though `is` was equivelant to `==` rather than just an alias of `is(equalTo())`. Will use in future!